### PR TITLE
Add new tarball to OBS

### PR DESCRIPTION
**Pull request by toabctl**
If a version update happened, in addition to removing old tarballs,
also add the new tarball to OBS.

### CODE REVIEW

File: renderspec [hunk 1]
```diff
@@ -116,19 +116,25 @@ def _get_spec_version(specfile):
     return list(set(filter(None, version.decode("utf-8").split('\n'))))[0]
 
 
-def _obs_remove_old_version_source(old_spec_version):
+def _obs_remove_add_version_source(old_spec_version, new_spec_version):
```
Comment (aplanas) [inline renderspec]: _obs_update_source?

File: renderspec [hunk 2]
```diff
@@ -116,19 +116,25 @@ def _get_spec_version(specfile):
...
-                                                  ','.join(rm_extensions))]
+                                                  ','.join(extensions))]
     with open(os.devnull, 'w') as devnull:
         subprocess.call(' '.join(osc_rm), stdout=devnull,
                         stderr=subprocess.STDOUT, shell=True)
 
+    osc_add = [OSC_BIN, 'add', '*-%s.{%s}' % (new_spec_version,
+                                              ','.join(extensions))]
+    with open(os.devnull, 'w') as devnull:
+        subprocess.call(' '.join(osc_add), stdout=devnull,
```
Comment (aplanas) [inline renderspec:135]: I think that you can avoid the .join() and pass the array directly

**Review by aplanas — APPROVED** [review]
LGTM, only some comments

File: renderspec (hunk 1 above)
Comment (dirkmueller) [reply renderspec]: I'd prefer addremove or add_remove which is similar to the dreaded osc command

Comment (toabctl) [reply renderspec]: done

File: renderspec (hunk 2 above)
Comment (toabctl) [reply renderspec:135]: iirc it does not work because I use shell=True

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**